iambic/config/wizard.py

```diff
@@ -418,6 +387,43 @@ def cf_role_arn(self):
 
         return self._cf_role_arn
 
+    @property
+    def has_aws_account_or_organizations(self) -> bool:
+        return self.config.aws and (
+            self.config.aws.accounts or self.config.aws.organizations
+        )
+
+    @property
+    def secrets_message(self) -> tuple[str, bool]:
+        """Returns if secrets are configured"""
+        secret_in_config = bool(self.config.extends)
+
+        if (
+            secret_in_config
+            and self.config.extends[0].key == ExtendsConfigKey.AWS_SECRETS_MANAGER
+        ):
+            secret_question_text = (
+                "This requires the ability to update the AWS Secrets Manager secret."
+            )
+        elif (
+            secret_in_config
+            and self.config.extends[0].key == ExtendsConfigKey.LOCAL_FILE
+        ):
+            secret_question_text = (
+                "This requires permissions to update secrets to you secrets file."
+            )
+        else:
+            secret_question_text = "Secrets must be created."
+
+        return (secret_question_text, secret_in_config)
+
+    @property
+    def has_okta_configured(self) -> bool:
+        return self.config.okta and self.config.okta.organizations
+
+    def secret_config_dir(self, secrets_file: str = "secrets.yaml") -> str:
+        return f"{Path(self.config_path).parent}/{secrets_file}"
```

Comment:
It might be easier to read using `joinpath` versus `/` in this case as the slash is easily overlooked between the curly braces.